CONTRIBUTING.md

```diff
@@ -1,8 +1,7 @@
 # Contributing
 
 We love pull requests from everyone. By participating in this project, you
-agree to abide by the [Contributor Covenant](http://contributor-covenant.org).
-
+agree to abide by the [Code Of Conduct](https://github.com/EddieHubCommunity/.github/blob/main/CODE_OF_CONDUCT.md).
 ## Steps
```

Comment:
idk what happened but you didn't had to remove both the lines 😅 

```suggestion

## Steps
```

Reply:
ooof, this might be my most confusing PR yet 🤦 , lemme know about any more changes (which I'm sure there are)